test/runAll.unmocked.spec.js

```diff
@@ -119,6 +114,11 @@ describe('runAll', () => {
     console = globalConsoleTemp
   })
 
+  it('should short-circuit with no staged files', async () => {
```

Comment:
I don’t like the name of the test. It made me think of electricity and there it has a bad connotation

Reply:
Sure, let’s call _exiting early_ or something.